src/uu/dd/src/dd.rs

```diff
@@ -80,8 +80,7 @@ impl Input<io::Stdin> {
         };
 
         if let Some(amt) = skip {
-            let mut buf = vec![BUF_INIT_BYTE; amt];
-            i.force_fill(&mut buf, amt)
+            i.read_skip(amt)
```

Comment:
FYI this will conflict with #3066.

Reply:
Thanks for the heads-up, I will politely wait for that to merge and fix whatever conflicts will arise